src/hooks/useAudioVolume.ts

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useRef, useState } from "react";
+import Bugsnag from "@bugsnag/js";
+
+import { MAX_AUDIO_VOLUME, MIN_AUDIO_VOLUME } from "settings";
+
+export const useAudioVolume: (options: {
+  audioElement?: HTMLAudioElement;
+  initialVolume: number;
+  isAudioPlaying: boolean;
+  storageKey: string;
+}) => {
+  volume: number;
+  setVolume: (volume: number) => void;
+} = ({ audioElement, initialVolume, isAudioPlaying, storageKey }) => {
+  const [volume, setVolume] = useState<number>(initialVolume);
```

Comment:
should we use `useVolumeControl` here?

Reply:
I was asking myself the same questions, but was hesitant to remove stuff before I add the minimal required functionality for this PR